newswires/app/db/FingerpostWireEntry.scala

```diff
@@ -407,7 +415,11 @@ object FingerpostWireEntry
 
     val maybeSinceId = queryParams.maybeSinceId
 
-    val highlightsClause = buildHighlightsClause(queryParams.maybeSearchTerm)
+    val highlightsClause = buildHighlightsClause(
+      queryParams.maybeSearchTerm.find(term =>
+        term.searchConfig == SearchConfig.English
```

Comment:
Small point, but it might be better to do a type check on the search term rather than an equality check, since I'm not 100% whether this will work since 
SearchConfig.English takes in values. An AI bot might be able to validate that haha. 
but I think: 
```
queryParams.maybeSearchTerm.collect({case e: SearchConfig.English => e}).heapOption
```
should do what we want here.